velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1399,4 +1400,16 @@ struct TimeZoneMinuteFunction : public TimestampWithTimezoneSupport<T> {
   }
 };
 
+template <typename T>
+struct ToISO8601FromDateFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<Varchar>& result,
+      const arg_type<Date>& date) {
+    auto dateType = DateType::get();
+    result = dateType->toString(date);
```

Comment:
Do you need local variable for dateType ? I think DATE()->toString(...) should work.

Reply:
Didn't realize there was a wrapper for `DateType::get()`. Replaced, thanks.

Reply:
True, thanks. Replaced.